corehq/apps/reminders/dbaccessors.py

```diff
@@ -0,0 +1,9 @@
+def get_surveys_in_domain(domain):
+    from corehq.apps.reminders.models import Survey
+    return sorted(Survey.view(
```

Comment:
The old view sorted by survey name (by emitting [domain, name]); since the new view doesn't, we sort it in memory to achieve the same effect.

For more of a blow by blow see https://github.com/dimagi/commcare-hq/commit/7c870d3b9a8c44877a66d17e2c74cac4954332fe and https://github.com/dimagi/commcare-hq/commit/4efd6257e76288a55df95dba47c742126c8a2a0f